lib/util/equation_processor.dart

```diff
@@ -1,117 +1,93 @@
 import "dart:collection";
 
-import 'calculation_handler.dart';
-import 'dartulator_strings.dart';
+import 'calculation.dart'
+    show Calculation, UnaryCalculation, BinaryCalculation;
+import 'dartulator_strings.dart' show parenthesis_open, parenthesis_close;
 
 class EquationProcessor {
 
   // modified from https://www.tutorialspoint.com/javaexamples/data_intopost.htm
-  static process(List input) {
-    Queue<String> operators = new Queue();
+  static process(List equationList) {
+    Queue calculations = new Queue();
```

Comment:
Hmm.... Mixed object Queues aren't ideal. You know, as a Java programmer, I need typed _everything..._

As it stands, changing how parenthesis are done is out-of-scope, but it should be considered at some point in the future.